### DISCUSSION TITLE
갑자기 게시물이 깨지는 현상이 발생합니다

### DISCUSSION THREAD

**Discussion by shwoo03** [pronist/hELLO#223] [Q&A] [answered]
안녕하십니까 현재 제작해주신 티도리 스킨 너무 잘 사용하고 있습니다! 
오늘 게시물들이 갑자기 사진과 같이 깨지는 형식으로 나타납니다.
2개 카테고리와 전체 분류보기에서만 이런 현상이 발생하는데 혹시 도움 주실 수 있나요... ㅠㅠ
![스크린샷 2024-04-29 200709](https://github.com/pronist/hello/assets/99020376/77ba42c0-8b87-4fb8-b749-ded5f1ff7885)
![스크린샷 2024-04-29 200742](https://github.com/pronist/hello/assets/99020376/95dca88d-3980-4ac7-9aa7-f60d50f5e94d)

**Comment by pronist** [comment, marked as answer]
@shwoo03 해당 현상이 발생한 블로그 링크 남겨주시면 감사하겠습니다.

**Reply by kiseokkm** [reply]
안녕하세요 스킨 잘 쓰고 있습니다 선생님 !!
저도 갑자기 화면이 깨지는 현상이 나타납니다 2,3페이지는 잘 나오는데 1페이지는 깨지는 현상이 있습니다
블로그 주소 : https://kiseokkm.tistory.com/ 입니다 감사합니다.

**Reply by pronist** [reply]
@kiseokkm 혹시 어떤 글을 쓰셨거나, 아니면 낮에는 안 그랬는데 밤에 확인해보니 갑자기 그러신건가요?

**Reply by pronist** [reply]
관련 이슈에 대한 논의는 <https://github.com/pronist/hello/issues/224> 에서 살펴보시기 바랍니다.

**Reply by kiseokkm** [reply]
낮 밤 확인은 안해봤는데 게시글을 추가하니깐 갑자기 깨졌네요 ! 살펴보겠습니다 감사합니다
